ytdl_server.py

```diff
@@ -1,5 +1,21 @@
-import os, subprocess, sys, urlparse
-import BaseHTTPServer, SimpleHTTPServer, SocketServer
+#!/usr/bin/env python
+from __future__ import print_function
+
+import os
+import subprocess
+import sys
```

Comment:
Apologies, I didn't mean to include this in this PR - I found this as part of <https://github.com/scrooloose/syntastic> which did a stylecheck and recommended having them on separate lines; I would say it is to make sure that it is obvious what the imports are.